### ISSUE TITLE
In a Hurry snippet

### ISSUE THREAD

**Issue by bwarren2** [skadistats/skadi#20]
Does the "In a Hurry" section run?  Using a demo of mine and the test demo, I do not see ehandle or state, but could be missing something.  I am poking around the objects and not seeing those attributes.

```
with io.open('tests/data/test.dem', 'r+b') as infile:
  demo = rd.construct(infile)
  stream=demo.stream()
  for tick, string_tables, world in stream:
    print dir(world), world.items()
```

['**class**', '**delattr**', '**dict**', '**doc**', '**format**', '**getattribute**', '**hash**', '**init**', '**iter**', '**module**', '**new**', '**reduce**', '**reduce_ex**', '**repr**', '**setattr**', '**sizeof**', '**slotnames**', '**str**', '**subclasshook**', '**weakref**', 'by_cls', 'by_dt', 'by_ehandle', 'by_index', 'classes', 'create', 'delete', 'fetch_cls', 'fetch_recv_table', 'find', 'find_all_by_cls', 'find_all_by_dt', 'find_by_cls', 'find_by_dt', 'find_index', 'recv_tables', 'update']
Traceback (most recent call last):
  File "<stdin>", line 5, in <module>
AttributeError: 'World' object has no attribute 'items'

**Comment by andrewzwicky** [comment]
Try using:

``` python
with io.open('tests/data/test.dem', 'r+b') as infile:
    demo = rd.construct(infile)
    stream=demo.stream()
    for tick, modifiers, world in stream:
        for ehandle,state in world.by_ehandle.items():
            print ehandle, len(state)
```

edit: use `world.by_ehandle.items()`

**Comment by onethirtyfive** [comment]
Resolved by pull request accompanying issue #21.
